README.adoc

```diff
@@ -6,20 +8,70 @@ image:https://img.shields.io/badge/docs-rust-blue.svg?style=flat-square[Docs,lin
 image:https://img.shields.io/pypi/v/deltalake.svg?style=flat-square[Python binding,link=https://pypi.org/project/deltalake]
 image:https://img.shields.io/badge/docs-python-blue.svg?style=flat-square[Docs,link=https://delta-io.github.io/delta-rs/python]
 
-An experimental interface to
-link:https://delta.io[Delta Lake]
-for Rust.
+A native interface to
+link:https://delta.io[Delta Lake].
+
+toc::[]
 
-This library provides low level access to Delta tables and is intended to be
+== About
+
+This library provides low level access to Delta tables in Rust, which can be
 used with data processing frameworks like
 link:https://github.com/apache/arrow/tree/master/rust/datafusion[datafusion],
 link:https://github.com/ballista-compute/ballista[ballista],
 link:https://github.com/nevi-me/rust-dataframe[rust-dataframe],
-link:https://github.com/rajasekarv/vega[vega], etc.
+link:https://github.com/rajasekarv/vega[vega], etc. It also provides bindings to other higher level languages such as link:https://delta-io.github.io/delta-rs/python/[Python], Ruby, or Golang.
+
+=== Features
+
+**Supported backends:**
+
+* Local file system
+* AWS S3
+* Azure Data Lake Storage Gen 2
+* :x: Google Cloud Storage see link:https://github.com/delta-io/delta-rs/issues/56[#56]
+
+.Support features
+|===
+| Operation/Feature | Rust | Python | Ruby
+
+| Read table metadata
+| :heavy_check_mark: 
+| :heavy_check_mark: 
+| :heavy_check_mark:
 
-It can also act as the basis for native bindings in other languages such as Python, Ruby or Golang.
+| Filter files with partitions
+| :heavy_check_mark: 
+| :heavy_check_mark: 
+|
+
+| Vacuum (delete stale files)
+| link:https://github.com/delta-io/delta-rs/issues/97[#97]
+|
```

Comment:
we can add the same link for python as well since @fvaleye already added the wrapper, once 97 is merged, python binding will get the fully implementation automatically.

```suggestion
| link:https://github.com/delta-io/delta-rs/issues/97[#97]
```